core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.spec.ts

```diff
@@ -75,8 +75,8 @@ describe('Audio Bar Component', () => {
     fixture.detectChanges();
   });
   beforeEach(() => {
-    spyOn(audioBarStatusService, 'markAudioBarExpanded').and.returnValue(null);
-    spyOn(audioBarStatusService, 'markAudioBarCollapsed').and.returnValue(null);
+    spyOn(audioBarStatusService, 'markAudioBarExpanded').and.returnValue();
+    spyOn(audioBarStatusService, 'markAudioBarCollapsed').and.returnValue();
```

Comment:
Use `.and.callThrough();` instead of `.and.returnValue();`

Reply:
This change seems to cause one of the ChromeHeadless tests to fail. Are there any particular reasons why we should change to this function?

Reply:
Can you direct me to the error logs?

Reply:
Oh sorry, I just ran the tests again and they all passed - not sure if it was an internet problem that caused the test to fail. The new changes are now pushed.